squeasel.c

```diff
@@ -4375,9 +4376,25 @@ static int set_ssl_option(struct sq_context *ctx) {
   }
 #endif // NO_SSL_DL
 
-  // Initialize SSL library
-  SSL_library_init();
-  SSL_load_error_strings();
+  // Initialize SSL library, unless the user has disabled this.
+  if (sq_strcasecmp(ctx->config[SSL_GLOBAL_INIT], "yes") == 0) {
```

Comment:
Yea, the assumption is that the hosting app has to make sure they initialize SSL before they use squeasel